.github/workflows/publish.yaml

```diff
@@ -15,4 +15,4 @@ jobs:
       - uses: actions/checkout@v2
 
       - uses: imjasonh/setup-ko@v0.4
-      - run: ko publish ./
+      - run: ko publish --base ./
```

Comment:
I thought `-B` was the right flag or `--base-import-paths`. Is that wrong?

Reply:
ah, `--bare` was the flag I was trying to use.  `KO_DOCKER_REPO` is set automatically by the `setup-ko` action to be `ghcr.io/<org>/<repo>`